### PR TITLE
EWPP-1895: Update to PHP8.

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -30,22 +31,25 @@ services:
...
 pipeline:
-  composer-install:
+  composer-install-highest:
     group: prepare
     image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
     volumes:
       - /cache:/cache
     commands:
+      - composer self-update --2
       - composer install --ansi --no-suggest --no-progress
```
Comment (hernani) [inline .drone.yml]: Remove no-suggest

File: .drone.yml [hunk 2]
```diff
@@ -30,22 +31,25 @@ services:
...
-    group: post-prepare
+    group: prepare
     image: registry.fpfis.eu/fpfis/httpd-php:${PHP_VERSION}-ci
     volumes:
       - /cache:/cache
     commands:
-      # @todo remove "composer install" step once the following issue is fixed.
-      # @link https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1234
+      - composer self-update --2
       - composer update --prefer-lowest --prefer-stable --ansi --no-suggest --no-progress
```
Comment (hernani) [inline .drone.yml]: Remove no-suggest

File: .drone.yml (hunk 1 above)
Comment (yenyasinn) [reply .drone.yml]: done

**Review by hernani — APPROVED** [review]

File: composer.json [hunk 3]
```diff
@@ -41,8 +41,7 @@
     "autoload-dev": {
         "psr-4": {
             "Drupal\\Tests\\oe_corporate_countries\\": "./tests/src/",
-            "Drupal\\Tests\\oe_corporate_countries_address\\": "./modules/oe_corporate_countries_address/tests/src/",
-            "Drupal\\Tests\\rdf_skos\\": "./build/modules/contrib/rdf_skos/tests/src/"
```
Comment (22Alexandra) [inline composer.json:45]: Why was this removed?

Comment (22Alexandra) [inline composer.json:45]: Why was this removed?

**Review by 22Alexandra — CHANGES_REQUESTED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**